cbioportal_etl/scripts/get_study_metadata.py

```diff
@@ -136,7 +136,7 @@ def run_py(args):
 
     datasheet_dir = 'datasheets'
     # Load json config file with database pull info
-    with open(args.meta_config) as f:
+    with open(args.study_config) as f:
```

Comment:
I just tried this locally to update my files, and I get:
`AttributeError: 'Namespace' object has no attribute 'study_config'`.
I believe this is because in line 181, `dest="config_file",`. So, either change that to `study_config`, or vice versa please